### DISCUSSION TITLE
Allowing URLs from the site to be used on works listed on MusicBrainz

### DISCUSSION THREAD

**Discussion by andy5995** [Lyrics/lyrics#529] [General]
There's an option to add a link to a lyrics page on each work listed on MusicBrainz. It only allows URL's from sites that are on the [whitelist](https://musicbrainz.org/doc/Style/Relationships/URLs/Lyrics_whitelist).

I opened a ticket to propose this project's website

https://tickets.metabrainz.org/browse/STYLE-1501

**Comment by andy5995** [comment]
> There's an option to add a link to a lyrics page on each work listed on MusicBrainz. It only allows URL's from sites that are on the [whitelist](https://musicbrainz.org/doc/Style/Relationships/URLs/Lyrics_whitelist).
> 
> I opened a ticket to propose this project's website
> 
> https://tickets.metabrainz.org/browse/STYLE-1501

They've responded and closed the ticket.

https://tickets.metabrainz.org/browse/STYLE-1501?focusedCommentId=56944&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-56944

I also asked for clarification in [this forum post](https://community.metabrainz.org/t/copyright-rules-of-lyrics-website-on-the-whitelist/510826).

**Comment by snshn** [comment]
Hey there! Great idea, and thank you for contacting them! The copyright message was taken from genius.com, yes — a lawyer told me it's the best I can do to minimize the chance of it being taken down.

It's kind of weird, this falls under the fair use due to not being intended for commercial use but rather educational use (free speech and all that), yet a lot of entities and people will be too scared to touch anything that contains lyrics — just because things still get taken down that fall under fair use... just remember the story of youtube-dl being taken down; good lesson for us all regarding how these things may just happen one day to any repo.

Here lyrics are transcribed, hand-written, and polished by users, hence even if the song's lyrics belong to some author or publishing company, the project cannot attribute copyright to the text that's not 100% identical to where it was taken from (say, author's website). Even slightest modification would force us to state that "the original text, that this was inspired/based on, belongs to ..." hence that legal metadata key `Original text copyright`, and not `Text copyright`.

I'm not surprised at all by their response, but thanks again for proposing that idea!

**Reply by andy5995** [reply]
> Hey there! Great idea, and thank you for contacting them! The copyright message was taken from genius.com, yes — a lawyer told me it's the best I can do to minimize the chance of it being taken down.

You're welcome.

> 
> Here lyrics are transcribed, hand-written, and polished by users, hence even if the song's lyrics belong to some author or publishing company, the project cannot attribute copyright to the text that's not 100% identical to where it was taken from (say, author's website). Even slightest modification would force us to state that "the original text, that this was inspired/based on, belongs to ..." hence that legal metadata key `Original text copyright`, and not `Text copyright`.
> 

But did you see this? https://genius.com/static/licensing

>Genius Media Group, Inc. (GMG) is fully licensed to display lyrics across all of its properties. In 2013, GMG entered into licenses with every major music publisher: Sony/ATV Music Publishing, EMI Music Publishing, Universal Music Publishing Group, and Warner/Chappell Music. In addition, GMG developed a form license with the National Music Publishers' Association (NMPA) which today covers more than 96% of the independent publisher market.
